en/asgardeo/features.json

```diff
@@ -0,0 +1,19 @@
+{
+    "asgardeo_logs": {
+        "enabled": true,
+        "page": [
+            "guides/asgardeo-logs/index.md",
+            "guides/asgardeo-logs/diagnostic-logs.md",
+            "guides/asgardeo-logs/audit-logs.md"
+        ]
+    },
+
+    "applications": {
+        "enabled": true,
+        "page": [
+            "guides/applications/register-single-page-app.md",
+            "guides/applications/register-oidc-web-app.md",
+            "guides/applications/register-saml-web-app.md"
+        ]
+    }
+}
```

Comment:
Let's add a new line in the end

Reply:
fixed with https://github.com/wso2/docs-is/pull/4799/commits/d92ebb5644d0ecec8a87e308c4fdcabe9ee25c49